pkg/urbit/vere/saga.c

```diff
@@ -456,6 +522,21 @@ u3_saga_open(const c3_path* const pax_u, u3_meta* const met_u)
   return log_u;
 }
 
+u3_saga*
+u3_saga_open(const c3_path* const pax_u, u3_meta* const met_u)
+{
+  return _saga_open(pax_u, met_u, NULL);
+}
+
+c3_i
+u3_saga_sane(const c3_path* const pax_u)
+{
+  u3_meta met_u;
+  c3_i    san_i = 0;
+  _saga_open(pax_u, &met_u, &san_i);
+  return san_i;
+}
+
```

Comment:
Hah, actually had this implemented as `(1)` at first, but ended up deciding against it because it felt like a weird thing to do.

I can see your argument for not wanting random exit statuses. There are already non-sanity-checking cases where we return `NULL` though, I guess we're fine not being able to distinguish those in the do-sanity-check case?

Reply:
Ha, classic. The way I see it, `u3_saga_open()` should return `NULL` when the event log couldn't be opened, whether that's the result of a sanity check or not. I can't think of any cases where you'd want to do X when the event log couldn't be opened for a non sanity check reason and Y when the event log couldn't be opened because the sanity check failed. All that said, I think standardized return statuses would be a huge improvement to Vere, but I think to do so will require strategizing.